sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -315,30 +319,50 @@ void ins_int_propagate(struct Int32Vect3 *accel, float dt)
 
 static void baro_cb(uint8_t __attribute__((unused)) sender_id, float pressure)
 {
-  if (!ins_int.baro_initialized && pressure > 1e-7) {
-    // wait for a first positive value
-    ins_int.qfe = pressure;
-    ins_int.baro_initialized = true;
-  }
-
-  if (ins_int.baro_initialized) {
-    if (ins_int.vf_reset) {
-      ins_int.vf_reset = false;
-      ins_int.qfe = pressure;
-      vff_realign(0.);
-      ins_update_from_vff();
+  static uint8_t num_points = 0;
```

Comment:
Will do. Is this a good idea though? I had implemented this cause the baro on the bebop is HORRIBLY noisy and changing the certainty of the Kalman filter wasn't enough to filter it out. I can imagine that this may not be very beneficial for other baros. I think I may move this to only apply to the bebop baro.